suites/nautilus/rgw/tier-1_rgw_multisite_test-upgrade-4.x-to-latest.yaml

```diff
@@ -0,0 +1,395 @@
+              rgw_realm: USA
+              rgw_zonemaster: false
+              rgw_zonesecondary: true
+              rgw_zonegroupmaster: false
+              rgw_zone_user: synchronization-user
+              rgw_zone_user_display_name: "Synchronization User"
+              system_access_key: 86nBoQOGpQgKxh4BLMyq
+              system_secret_key: NTnkbmkMuzPjgwsBpJ6o
+              rgw_multisite_proto: "http"
+              rgw_pullhost: "{node_ip:ceph-rgw1#node5}"
+              rgw_pull_proto: http
+              rgw_pull_port: 8080
+              upgrade_ceph_packages: true
+      desc: Upgrading the clusters to the specified build.
+      abort-on-fail: true
+      polarion-id: CEPH-83574658
+
+  # Test cluster post upgrade
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_Mbuckets_with_Nobjects_delete.yaml
+            timeout: 300
+      desc: test to create "M" no of buckets and "N" no of objects with delete
+      module: sanity_rgw_multisite.py
+      name: Test delete using M buckets with N objects
+      polarion-id: CEPH-14237
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_Mbuckets_with_Nobjects_download.yaml
+            timeout: 300
+      desc: test to create "M" no of buckets and "N" no of objects with download
+      module: sanity_rgw_multisite.py
+      name: Test download with M buckets with N objects
+      polarion-id: CEPH-14237
+
+  # Basic swift based tests
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_swift_basic_ops.py
+            config-file-name: test_swift_version_copy_op.yaml
+            timeout: 500
+      desc: test restoring of versioned objects in swift
+      module: sanity_rgw_multisite.py
+      name: test swift versioning copy
+      polarion-id: CEPH-10646
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_swift_basic_ops.py
+            config-file-name: test_swift_object_expire_op.yaml
+            timeout: 500
+      desc: test object expiration with swift
+      module: sanity_rgw_multisite.py
+      name: test swift object expiration
+      polarion-id: CEPH-9718
+
+  # DBR
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_multisite_dynamic_resharding_greenfield.yaml
+            verify-io-on-site: ["ceph-rgw1", "ceph-rgw2"]
+            timeout: 300
+            test-config:
+              user_count: 1
+              bucket_count: 1
+              objects_count: 100
+              dynamic_resharding: true
+              objects_size_range:
+                min: 15K
+                max: 500K
+              cluster_type: greenfield
```

Comment:
Since this is 4.x upgrade suite, Instead of greenfield, we need to add brownfield(Not yet automated) scenario here

Reply:
I have removed them as discussed and let's add the brownfield one once we have the scripts merged